sdk/monitor/monitor-query/swagger/metric-Batch.md

```diff
@@ -0,0 +1,26 @@
+# Azure Monitor Batch TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5f700acd3d094d8eedca381932f2e7916afd2e55/specification/monitor/data-plane/Microsoft.Insights/preview/2023-05-01-preview/metricBatch.json
+output-folder: ../src/generated/metricBatch
+package-name: "monitor-metric-batch"
+package-version: "1.0.1"
```

Comment:
What is this version used for?

Reply:
Used during the generation process. To be consistent, I have modified the value here also. Will be available in the next commit